### PR TITLE
Remove repeated words (part 2)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #14572

### Summary
Found more repeated words and panctuations.

#### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/gamepadpose/angularacceleration/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ browser-compat: api.GamepadPose.angularAcceleration
 ---
 {{APIRef("WebVR API")}}{{SeeCompatTable}}
 
-The **`angularAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("Gamepad")}}, in meters per second per second.
+The **`angularAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("Gamepad")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepadpose/angularacceleration/index.md:18]: This was correct. An acceleration is expressed in m/s^2. That is: meters per second per second.
```suggestion
The **`angularAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("Gamepad")}}, in meters per second per second.
```

File: files/en-us/web/api/gamepadpose/index.md [hunk 2]
```diff
@@ -30,13 +30,13 @@ This interface is accessible through the {{domxref("Gamepad.pose")}} property.
 - {{domxref("GamepadPose.linearVelocity")}} {{readonlyInline}}
   - : Returns the linear velocity of the {{domxref("Gamepad")}}, in meters per second.
 - {{domxref("GamepadPose.linearAcceleration")}} {{readonlyInline}}
-  - : Returns the linear acceleration of the {{domxref("Gamepad")}}, in meters per second per second.
+  - : Returns the linear acceleration of the {{domxref("Gamepad")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepadpose/index.md:33]:
```suggestion
  - : Returns the linear acceleration of the {{domxref("Gamepad")}}, in meters per second per second.
```

File: files/en-us/web/api/gamepadpose/index.md [hunk 3]
```diff
@@ -30,13 +30,13 @@ This interface is accessible through the {{domxref("Gamepad.pose")}} property.
...
 - {{domxref("GamepadPose.orientation")}} {{readonlyInline}}
   - : Returns the orientation of the {{domxref("Gamepad")}}, as a quarternion value.
 - {{domxref("GamepadPose.angularVelocity")}} {{readonlyInline}}
   - : Returns the angular velocity of the {{domxref("Gamepad")}}, in radians per second.
 - {{domxref("GamepadPose.angularAcceleration")}} {{readonlyInline}}
-  - : Returns the angular acceleration of the {{domxref("Gamepad")}}, in meters per second per second.
+  - : Returns the angular acceleration of the {{domxref("Gamepad")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepadpose/index.md:39]:
```suggestion
  - : Returns the angular acceleration of the {{domxref("Gamepad")}}, in meters per second per second.
```

File: files/en-us/web/api/gamepadpose/linearacceleration/index.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ browser-compat: api.GamepadPose.linearAcceleration
 ---
 {{APIRef("WebVR API")}}{{SeeCompatTable}}
 
-The **`linearAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("Gamepad")}}, in meters per second per second.
+The **`linearAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("Gamepad")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/gamepadpose/linearacceleration/index.md:18]:
```suggestion
The **`linearAcceleration`** read-only property of the {{domxref("GamepadPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("Gamepad")}}, in meters per second per second.
```

File: files/en-us/web/api/vrpose/angularacceleration/index.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ browser-compat: api.VRPose.angularAcceleration
 ---
 {{APIRef("WebVR API")}}{{Deprecated_Header}}
 
-The **`angularAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
+The **`angularAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/vrpose/angularacceleration/index.md:18]:
```suggestion
The **`angularAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the angular acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
```

File: files/en-us/web/api/vrpose/index.md [hunk 6]
```diff
@@ -27,13 +27,13 @@ This interface is accessible through the {{domxref("VRDisplay.getPose()")}} and
 - {{domxref("VRPose.linearVelocity")}} {{deprecated_inline}}{{readonlyInline}}
   - : Returns the linear velocity of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second.
 - {{domxref("VRPose.linearAcceleration")}} {{deprecated_inline}}{{readonlyInline}}
-  - : Returns the linear acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
+  - : Returns the linear acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/vrpose/index.md:30]:
```suggestion
  - : Returns the linear acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
```

File: files/en-us/web/api/vrpose/index.md [hunk 7]
```diff
@@ -27,13 +27,13 @@ This interface is accessible through the {{domxref("VRDisplay.getPose()")}} and
...
 - {{domxref("VRPose.orientation")}} {{deprecated_inline}}{{readonlyInline}}
   - : Returns the orientation of the sensor at the current {{domxref("VRPose.timestamp")}}, as a quarternion value.
 - {{domxref("VRPose.angularVelocity")}} {{deprecated_inline}}{{readonlyInline}}
   - : Returns the angular velocity of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in radians per second.
 - {{domxref("VRPose.angularAcceleration")}} {{deprecated_inline}}{{readonlyInline}}
-  - : Returns the angular acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
+  - : Returns the angular acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/vrpose/index.md:36]:
```suggestion
  - : Returns the angular acceleration of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
```

File: files/en-us/web/api/vrpose/linearacceleration/index.md [hunk 8]
```diff
@@ -15,7 +15,7 @@ browser-compat: api.VRPose.linearAcceleration
 ---
 {{APIRef("WebVR API")}}{{Deprecated_Header}}
 
-The **`linearAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
+The **`linearAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second.
```
Comment (teoli2003) [inline files/en-us/web/api/vrpose/linearacceleration/index.md:18]:
```suggestion
The **`linearAcceleration`** read-only property of the {{domxref("VRPose")}} interface returns an array representing the linear acceleration vector of the {{domxref("VRDisplay")}} at the current {{domxref("VRPose.timestamp")}}, in meters per second per second.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
@teoli2003 done

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**